shared/tasks/browser.mjs

```diff
@@ -56,6 +68,31 @@ export async function screenshots () {
     await Promise.all(screenshotTasks)
   }
 
+  // Extra 'manual' set of screenshots for rebrand examples
```

Comment:
Is there any way we could push these into the `input` array instead, so we can take advantage of the batching up of screenshots that's going on?

Feels like we've done most of the work to let us use `screenshotComponent` – I think all that would need to be done is allow for a third `options` item in the entries of the `input` array?

Reply:
Let me give this another go. I initially had some trouble with this which led me to cut some corners but in theory I can do something clever passing options in as an extra array index.

Reply:
Ok I think I have something operable now. It's a bit cheeky but it _works_. It's passing in the component with the example as a string to build (`[component]/[example]`) which is then built in `goToComponent`. This is as opposed to using `options.exampleName` in `goToComponent` which is why it feels cheeky.

Too cheeky?

Reply:
Actually looking at the percy output this isn't gonna work. It hasn't captured js and non-js versions of the components when they're component and example. I've got an idea of how to resolve this though. Watch this space.